client/src/Api.elm

```diff
@@ -167,7 +167,7 @@ mkAllConstructs url expect auth =
                 authenticatedGet usr.token (url ++ "/admin/constructs") expect
 
             else
-                Cmd.none
+                authenticatedGet usr.token (url ++ "/vectors/") expect
```

Comment:
I don't think you mean this. What are you using this for?

Reply:
I'll change it back.
I thought that if the user wasn't admin that the vectors for a certain user should fetched but now I see that mkAllConstructs is only used for the admin console.
I just got a bit confused with the name.